### PR TITLE
Styling event views, navbar linking

### LINKED ISSUE #58: Styling Event Views

### PR DESCRIPTION
**Pull request by bpietraga**
Closes #58, #61

### CODE REVIEW

**Comment by KamilFilip** [comment]
Looks fine +1

File: app/views/events/index.html.slim [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
-        td= link_to 'Destroy', event, method: :delete, data: { confirm: 'Are you sure?' }
-br/
-= link_to 'New Event', new_event_path
+- @events.each do |event|
+    .container
+      div class="padding-quarter"
+        div class="box bg-white img-rounded"
+          .grid.grid-pair
+            .grid-item class="4/4"
+                  div class="grid grid-pair grid-middle"
```
Comment (Snieezy) [inline app/views/events/index.html.slim]: wrong indentation?

File: app/views/events/show.html.slim [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
-= link_to 'Back', events_path
+.container
+  div class="box bg-white img-rounded"
+    h2.no-margin-top = @event.name
+    small Description:
+    strong
+      .text-field = @event.description
+    small City:
+    strong
+        .text-field = @event.city
```
Comment (Snieezy) [inline app/views/events/show.html.slim]: wrong indentation?

**Comment by maciek1995** [comment]
+1

**Comment by hav3loc** [comment]
Ready to be checked +1

**Comment by gotar** [comment]
why you have changed schema file inside your commit (commit about new view?)

I guess someone else before forgot to run migration?

**Comment by gotar** [comment]
code itself looks ok, you can merge, but you should pick other then FR tickets too. To not to close only for view layer.

**Pull request merged**